runtime/basilisk/src/lib.rs

```diff
@@ -906,62 +918,62 @@ construct_runtime!(
+		Tips: pallet_tips = 13,
+
+		Authorship: pallet_authorship exclude_parts { Inherent } = 14,
+		CollatorSelection: pallet_collator_selection = 15,
+		Session: pallet_session = 16, // Session must be after collator and before aura
+		Aura: pallet_aura exclude_parts { Storage } = 17,
+		AuraExt: cumulus_pallet_aura_ext exclude_parts { Storage } = 18,
+		Preimage: pallet_preimage = 19,
+		Uniques: pallet_uniques = 20,
+		Identity: pallet_identity = 21,
+		Multisig: pallet_multisig = 22,
 
 		// Parachain and XCM - starts at index 50
-		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Storage, Inherent, Event<T>, ValidateUnsigned} = 50,
-		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 51,
+		ParachainSystem: cumulus_pallet_parachain_system exclude_parts { Config } = 50,
+		ParachainInfo: parachain_info = 51,
 
-		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 52,
-		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin} = 53,
-		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Storage, Event<T>} = 54,
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 55,
+		PolkadotXcm: pallet_xcm = 52,
+		CumulusXcm: cumulus_pallet_xcm = 53,
+		XcmpQueue: cumulus_pallet_xcmp_queue exclude_parts { Call } = 54,
+		DmpQueue: cumulus_pallet_dmp_queue = 55,
 
 		// Basilisk - runtime module index for basilisk's pallets starts at 100
-		AssetRegistry: pallet_asset_registry::{Pallet, Call, Config<T>, Storage, Event<T>} = 100,
-		XYK: pallet_xyk::{Pallet, Call, Storage, Event<T>} = 101,
-		Duster: pallet_duster::{Pallet, Call, Config<T>, Storage, Event<T>} = 102,
-		Exchange: pallet_exchange::{Pallet, Call, Storage, Event<T>} = 103,
-		LBP: pallet_lbp::{Pallet, Call, Storage, Event<T>} = 104,
-		NFT: pallet_nft::{Pallet, Call, Event<T>, Storage} = 105,
-
-		MultiTransactionPayment: pallet_transaction_multi_payment::{Pallet, Call, Config<T>, Storage, Event<T>} = 106,
-		RelayChainInfo: pallet_relaychain_info::{Pallet, Event<T>} = 108,
-		Marketplace: pallet_marketplace::{Pallet, Call, Event<T>, Storage} = 109,
-		TransactionPause: pallet_transaction_pause::{Pallet, Call, Event<T>, Storage} = 110,
-		Router: pallet_route_executor::{Pallet, Call, Event<T>} = 111,
+		AssetRegistry: pallet_asset_registry = 100,
+		XYK: pallet_xyk = 101,
+		Duster: pallet_duster = 102,
+		Exchange: pallet_exchange = 103,
+		LBP: pallet_lbp = 104,
+		NFT: pallet_nft = 105,
+
+		MultiTransactionPayment: pallet_transaction_multi_payment = 106,
+		RelayChainInfo: pallet_relaychain_info = 108,
+		Marketplace: pallet_marketplace = 109,
+		TransactionPause: pallet_transaction_pause = 110,
+		Router: pallet_route_executor = 111,
 
 		// ORML related modules - runtime module index for orml starts at 150
-		Currencies: orml_currencies::{Pallet, Call, Event<T>} = 150,
-		Tokens: orml_tokens::{Pallet, Storage, Call, Event<T>, Config<T>} = 151,
+		Currencies: pallet_currencies = 150,
```

Comment:
@lumir-mrkva do we want to keep this pallet here to keep the same ID or to move it where the warehouse pallets are listed and reserve new ID for it?

Reply:
please keep the same ID to ensure compatibility - everything is encoded based on this ID